features/admins/admin_post_faqs.feature

```diff
@@ -2,85 +2,88 @@
-    When I press "Post"
+      And I press "Post"
     Then I should see "ArchiveFaq was successfully created"
-      And 0 emails should be delivered
 
-    
+    # Translate FAQ to "de"
+    When I follow "Archive FAQ"
+      And I select "Deutsch" from "language_id"
+      And I press "Go" within "div#inner.wrapper"
+      And I follow "Edit"
+      And I fill in "Question*" with "Was ist AO3?"
+      And I fill in "Answer*" with "Einige Text, das ist lang genug, um die Überprüfung bestanden."
+      And I fill in "Category name*" with "Neuer Abschnitt"
+      And I check "Question translated"
+      And I press "Post"
+    Then I should see "ArchiveFaq was successfully updated."
+      And I should not see "New subsection"
+      And I should see "Neuer Abschnitt"
+      And I should see "Was ist AO3?"
+      And I should see "Einige Text"
+
+    # Change locale "de" to "ger"
+    When I go to the locales page
+      And I follow "Edit"
+    Then I should see "Deutsch" in the "Name" input
+    When I fill in "locale_iso" with "ger"
+      And I press "Update Locale"
+    Then I should see "Your locale was successfully updated."
+      And I should see "Deutsch ger"
+
+    # The current session preference remains "de", the category name falls back to the default locale,
+    # but no "de" questions will be found.
+    When I go to the archive_faqs page
+      And I follow "New subsection"
+    Then I should not see "What is AO3?"
+      And I should not see "Was ist AO3?"
 
-    # Now post a Translation of that FAQ
-    # DISABLED UNTIL SOMEONE CAN FIGURE OUT HOW TO MAKE LANGUAGE SELECTION WORK
-    And "Language Selection" is fixed
-#    Given all emails have been delivered
-#    When I follow "Archive FAQ"
-#      And I choose "Deutsch" from "language_id"
-#      And I press "Go" within "div#inner.wrapper"
-#      And show me the page
-#      And I should see "New subsection"
-#      And I follow "Edit"
-#      And I fill in "Question*" with "Was ist AO3?"
-#      And I fill in "Answer*" with "Einige Text, das ist lang genug, um die Überprüfung bestanden."
-#      And I fill in "Category name*" with "Neuer Abschnitt"
-#      And I fill in "Anchor name*" with "wasistao3"
-#      And I press "Post"
-#    Then I should see "ArchiveFaq was successfully updated."
-#
-#    # The user has previously selected German as their language, lets make sure it persisted through Controller actions
-#    Then I should see "Questions in the Neuer Abschnitt Category"
-#      And I should not see "New subsection"
-#      And I should see "Was ist AO3?"
-#
-#    # Toggle the languages at the top and see the correct data
-#    When I follow "Archive FAQ"
-#      And I select "English" from "language_id"
-#      And I press "Go" within "div#inner.wrapper"
-#    Then I should see "New subsection"
-#      And I should not see "Neuer Abschnitt"
+    # Log out and view FAQs; the default locale should be used
+    When I log out
+      And I go to the archive_faqs page
```

Comment:
Without the fix (`after_update` for locales), the test dies here as described in https://otwarchive.atlassian.net/browse/AO3-4311?focusedCommentId=357975.